### DISCUSSION TITLE
erro  ao emitir NFCe - odoo 14

### DISCUSSION THREAD

**Discussion by gilsonlima10** [OCA/l10n-brazil#3157] [Q&A]
quando mando emitir NFCe no ponto de venda do odoo 14 , o sistema tras a seguinte mensagem de erro:  "sem processador localizado"

**Comment by antoniospneto** [comment]
@gilsonlima10 confere na configuração da empresa, lá tem um lugar que tem que definir o processador de documentos, tem que estar erpbrasil.edoc, assim:
![image](https://github.com/OCA/l10n-brazil/assets/634278/ef7abdc0-5000-43d0-b628-fe2782097311)

**Reply by gilsonlima10** [reply]
esta configurado desta forma!

**Comment by mileo** [comment]
você já conseguiu emitir pelo backend primeiro?

**Comment by gilsonlima10** [comment]
resolvido

**Reply by douglascstd** [reply]
Gilson, compartilha o seu aprendizado para manter registrado aqui...  ;-)

**Reply by gilsonlima10** [reply]
ainda não emitir nota fiscal com sucesso, mas vi que tinha cometido erro na configuração, estou evoluindo na configuração para ter o modulo fiscal funcionando em meu computador..agradeço o apoio.
